service/collector/elb.go

```diff
@@ -139,31 +143,77 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 
 	var lbs []loadBalancer
 	{
-		i := &elb.DescribeTagsInput{}
-		for _, l := range loadBalancers {
-			i.LoadBalancerNames = append(i.LoadBalancerNames, l.LoadBalancerName)
+		// AWS API has a limit for maximum number of LoadBalancerNames in
+		// single Describe request so it must be done in batches of
+		// maxELBsInOneBatch. In order to not spend so much time on this,
+		// perform requests concurrently and synchronize them with errgroup.
+		errGroup, _ := errgroup.WithContext(context.Background())
```

Comment:
```suggestion
		errGroup, _ := errgroup.WithContext(context.TODO())
```

Should be taken care of in https://github.com/giantswarm/giantswarm/issues/4909